docs/usage.md

```diff
@@ -805,3 +806,32 @@ moco_cluster_healthy{name="test",namespace="default"} NaN
 moco_cluster_ready_replicas{name="test",namespace="default"} NaN
 moco_cluster_errant_replicas{name="test",namespace="default"} NaN
 ```
+
+### Set to Read Only
+
+When you want to set MOCO's MySQL to read-only, use the the following commands.
+
+MOCO makes the primary instance writable in the clustering process.
+Therefore, please be sure to stop clustering when you set it to read-only.
+
+```console
+$ kubectl moco stop clustering <CLSUTER_NAME>
+$ kubectl moco mysql -u moco-admin <CLSUTER_NAME> -- -e "SET GLOBAL super_read_only=1"
+```
+
+You can check whether the cluster is read-only with the following command.
+
+```console
+$ kubectl moco mysql -it <CLSUTER_NAME> -- -e "SHOW GLOBAL VARIABLES like 'super_read_only'"
```

Comment:
nit. I usually use `SELECT @@super_read_only;`.

Reply:
Thank you. I updated. ce6332b